recipes/default.rb

```diff
@@ -24,10 +24,21 @@
 # ::Chef::Recipe.send(:include, Chef::Mixin::PowershellOut)
 ::Chef::Resource::RubyBlock.send(:include, Chef::Mixin::PowershellOut)
 
-powershell_script 'install chocolatey' do
-  code "iex ((new-object net.webclient).DownloadString('#{node['chocolatey']['Uri']}'))"
-  convert_boolean_return true
-  not_if { ChocolateyHelpers.chocolatey_installed? }
+if File.exist?('C:\windows\system32\cmd.exe')
+  arch = :x86_64
+  cmd = 'C:\windows\system32\cmd.exe'
```

Comment:
Okay, yeah I was going to update this that I had some questions around it.

What if someone has installed chef on their own 64-bit ruby (although technically this is not something we support) and so we're invoking from a 64-bit process on a 64-bit system?  Will the sysnative path still exist and still point at 64-bit?